### PR TITLE
Feature getby

### PR DESCRIPTION
**Pull request by mileszander**

### CODE REVIEW

**Comment by mileszander** [comment]
This is to pull my db data by id, also populated my db with dummy data

File: db/userSchema.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose')
+const connectionString = "mongodb+srv://root:root@reviews-y8owi.mongodb.net/test?retryWrites=true&w=majority"
```
Comment (rcalimlim) [inline db/userSchema.js:2]: uh oh

**Review by rcalimlim — CHANGES_REQUESTED** [review]
Should probably remove your db pass since you're already using .dotenv

File: db/userSchema.js (hunk 1 above)
Comment (mileszander) [reply db/userSchema.js:2]: :D

Comment (mileszander) [reply db/userSchema.js:2]: Updated to be super secure

**Pull request merged**